x-pack/plugins/spaces/public/copy_saved_objects_to_space/components/copy_mode_control.tsx

```diff
@@ -0,0 +1,174 @@
+      'xpack.spaces.management.copyToSpace.copyModeControl.includeRelated.text',
+      {
+        defaultMessage:
+          'This will copy any other objects this has references to -- for example, a dashboard may have references to multiple visualizations.',
+      }
+    ),
+  };
+  const copyOptionsTitle = i18n.translate(
+    'xpack.spaces.management.copyToSpace.copyModeControl.copyOptionsTitle',
+    { defaultMessage: 'Copy options' }
+  );
+  const relationshipOptionsTitle = i18n.translate(
+    'xpack.spaces.management.copyToSpace.copyModeControl.relationshipOptionsTitle',
+    { defaultMessage: 'Relationship options' }
+  );
+
+  const createLabel = ({ text, tooltip }: { text: string; tooltip: string }) => (
+    <EuiFlexGroup>
+      <EuiFlexItem>
+        <EuiText>{text}</EuiText>
+      </EuiFlexItem>
+      <EuiFlexItem grow={false}>
+        <EuiIconTip content={tooltip} position="left" type="iInCircle" />
+      </EuiFlexItem>
+    </EuiFlexGroup>
+  );
+  const onChange = (partial: Partial<CopyMode>) => {
+    if (partial.createNewCopies !== undefined) {
+      setCreateNewCopies(partial.createNewCopies);
+    } else if (partial.overwrite !== undefined) {
+      setOverwrite(partial.overwrite);
+    }
+    updateSelection({ createNewCopies, overwrite, ...partial });
+  };
+
+  return (
+    <>
+      <EuiFormFieldset
+        legend={{
+          children: (
+            <EuiTitle size="xs">
+              <span>{copyOptionsTitle}</span>
+            </EuiTitle>
+          ),
+        }}
+      >
+        <EuiCheckableCard
+          id={createNewCopiesDisabled.id}
+          label={createLabel(createNewCopiesDisabled)}
+          checked={!createNewCopies}
+          onChange={() => onChange({ createNewCopies: false })}
+        >
+          <EuiRadioGroup
+            options={[overwriteEnabled, overwriteDisabled]}
+            idSelected={overwrite ? overwriteEnabled.id : overwriteDisabled.id}
+            onChange={(id: string) => onChange({ overwrite: id === overwriteEnabled.id })}
+            disabled={createNewCopies}
+            data-test-subj={'cts-copyModeControl-overwriteRadioGroup'}
+          />
+        </EuiCheckableCard>
+
+        <EuiSpacer size="s" />
+
+        <EuiCheckableCard
+          id={createNewCopiesEnabled.id}
+          label={createLabel(createNewCopiesEnabled)}
+          checked={createNewCopies}
+          onChange={() => onChange({ createNewCopies: true })}
+        />
+      </EuiFormFieldset>
+
+      <EuiSpacer size="m" />
+
+      <EuiFormFieldset
+        legend={{
+          children: (
+            <EuiTitle size="xs">
+              <span>{relationshipOptionsTitle}</span>
+            </EuiTitle>
+          ),
+        }}
+      >
+        <EuiCheckableCard
+          id={includeRelated.id}
+          label={createLabel(includeRelated)}
+          checkableType="checkbox"
+          checked={true}
+          onChange={() => {}} // noop
+          disabled
+        />
```

Comment:
What is the role of this `EuiCheckableCard` if it can't be toggled/activated?

Reply:
It makes more sense when you look at the before and after:

![image](https://user-images.githubusercontent.com/5295965/90572951-3dadff00-e183-11ea-9438-eb9338babc18.png)

![image](https://user-images.githubusercontent.com/5295965/90573109-9aa9b500-e183-11ea-9eba-911f9bc29db3.png)

The "Include related saved objects" is there mostly as a placeholder / indicator that the user is not _only copying one object_. You can't disable it yet, but we plan to potentially add support for that someday.

When we changed the "Overwrite" option and split it out into the radios, the checkmark with a label looked out of place. Hence we decided to convert that to an `EuiCheckableCard` so the whole UI looked more consistent. But it can't be changed, so it's disabled.

Perhaps the tooltip should be updated to reflect that? I'll make a note to bring it up during our tech writer / design review in the final PR.